.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
What does this do?

Reply:
https://github.com/financial-times-sandbox/tako/pull/15/commits/47c58298241d130fc43147d6fb88d79574a46b4f

Reply:
Have added a comment to `.npmrc` to reference the npm config docs.